### ISSUE TITLE
Catkin make doesnt work

### ISSUE THREAD

**Issue by Plzr** [a-price/reactor_controller#3]
Hi,

I think my problems are non related (though both have risen because I use kinetic, I think), so I decided to make two issues. If I try catkin_make, it doesnt work. I get the following reply. This also happens when I try to make it using -j2. Any ideas?

```
remy-xubuntu@remy-xubuntu:~/catkin_a_price$ catkin_make
Base path: /home/remy-xubuntu/catkin_a_price
Source space: /home/remy-xubuntu/catkin_a_price/src
Build space: /home/remy-xubuntu/catkin_a_price/build
Devel space: /home/remy-xubuntu/catkin_a_price/devel
Install space: /home/remy-xubuntu/catkin_a_price/install
####
#### Running command: "make cmake_check_build_system" in "/home/remy-xubuntu/catkin_a_price/build"
####
####
#### Running command: "make -j4 -l4" in "/home/remy-xubuntu/catkin_a_price/build"
####
[  0%] Built target _dynamixel_controllers_generate_messages_check_deps_TorqueEnable
[  0%] Built target _dynamixel_controllers_generate_messages_check_deps_SetCompliancePunch
[  0%] Built target _dynamixel_controllers_generate_messages_check_deps_SetSpeed
[  0%] Built target _dynamixel_controllers_generate_messages_check_deps_SetComplianceSlope
[  0%] Built target _dynamixel_controllers_generate_messages_check_deps_RestartController
[  0%] Built target _dynamixel_controllers_generate_messages_check_deps_SetComplianceMargin
[  0%] Built target _dynamixel_controllers_generate_messages_check_deps_StopController
[  0%] Built target _dynamixel_controllers_generate_messages_check_deps_StartController
[  0%] Built target std_msgs_generate_messages_nodejs
[  0%] Built target _dynamixel_msgs_generate_messages_check_deps_JointState
[  0%] Built target _dynamixel_controllers_generate_messages_check_deps_SetTorqueLimit
[  0%] Built target _dynamixel_msgs_generate_messages_check_deps_MotorStateList
[  0%] Built target std_msgs_generate_messages_py
[  0%] Built target std_msgs_generate_messages_eus
[  0%] Built target _dynamixel_msgs_generate_messages_check_deps_MotorState
[  0%] Built target std_msgs_generate_messages_cpp
[ 27%] Built target dynamixel_controllers_generate_messages_nodejs
[ 27%] Built target dynamixel_controllers_generate_messages_cpp
[ 27%] Built target std_msgs_generate_messages_lisp
[ 42%] Built target dynamixel_controllers_generate_messages_py
[ 57%] Built target dynamixel_controllers_generate_messages_eus
[ 71%] Built target dynamixel_controllers_generate_messages_lisp
[ 75%] Built target dynamixel_msgs_generate_messages_nodejs
[ 80%] Built target dynamixel_msgs_generate_messages_cpp
[ 86%] Built target dynamixel_msgs_generate_messages_py
[ 92%] Built target dynamixel_msgs_generate_messages_eus
[ 92%] Built target dynamixel_controllers_generate_messages
[ 96%] Built target dynamixel_msgs_generate_messages_lisp
[ 96%] Built target dynamixel_msgs_generate_messages
[ 98%] Linking CXX executable /home/remy-xubuntu/catkin_a_price/devel/lib/reactor_controller/dynamixel_to_joint_state
CMakeFiles/dynamixel_to_joint_state.dir/src/dynamixel_to_joint_state.cpp.o: In function `XmlRpc::XmlRpcValue::operator[](char const*)':
/opt/ros/kinetic/include/XmlRpcValue.h:101: undefined reference to `XmlRpc::XmlRpcValue::assertStruct()'
CMakeFiles/dynamixel_to_joint_state.dir/src/dynamixel_to_joint_state.cpp.o: In function `XmlRpc::XmlRpcValue::operator int&()':
/opt/ros/kinetic/include/XmlRpcValue.h:91: undefined reference to `XmlRpc::XmlRpcValue::assertTypeOrInvalid(XmlRpc::XmlRpcValue::Type)'
CMakeFiles/dynamixel_to_joint_state.dir/src/dynamixel_to_joint_state.cpp.o: In function `XmlRpc::XmlRpcValue::operator[](char const*)':
/opt/ros/kinetic/include/XmlRpcValue.h:101: undefined reference to `XmlRpc::XmlRpcValue::assertStruct()'
CMakeFiles/dynamixel_to_joint_state.dir/src/dynamixel_to_joint_state.cpp.o: In function `XmlRpc::XmlRpcValue::operator double&()':
/opt/ros/kinetic/include/XmlRpcValue.h:92: undefined reference to `XmlRpc::XmlRpcValue::assertTypeOrInvalid(XmlRpc::XmlRpcValue::Type)'
CMakeFiles/dynamixel_to_joint_state.dir/src/dynamixel_to_joint_state.cpp.o: In function `XmlRpc::XmlRpcValue::operator[](std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&)':
/opt/ros/kinetic/include/XmlRpcValue.h:100: undefined reference to `XmlRpc::XmlRpcValue::assertStruct()'
/opt/ros/kinetic/include/XmlRpcValue.h:100: undefined reference to `XmlRpc::XmlRpcValue::assertStruct()'
CMakeFiles/dynamixel_to_joint_state.dir/src/dynamixel_to_joint_state.cpp.o: In function `dmxCallback(boost::shared_ptr<dynamixel_msgs::MotorStateList_<std::allocator<void> > > const&)':
/home/remy-xubuntu/catkin_a_price/src/reactor_controller/src/dynamixel_to_joint_state.cpp:72: undefined reference to `XmlRpc::XmlRpcValue::hasMember(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&) const'
CMakeFiles/dynamixel_to_joint_state.dir/src/dynamixel_to_joint_state.cpp.o: In function `XmlRpc::XmlRpcValue::operator[](std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&)':
/opt/ros/kinetic/include/XmlRpcValue.h:100: undefined reference to `XmlRpc::XmlRpcValue::assertStruct()'
CMakeFiles/dynamixel_to_joint_state.dir/src/dynamixel_to_joint_state.cpp.o: In function `XmlRpc::XmlRpcValue::operator[](char const*)':
/opt/ros/kinetic/include/XmlRpcValue.h:101: undefined reference to `XmlRpc::XmlRpcValue::assertStruct()'
CMakeFiles/dynamixel_to_joint_state.dir/src/dynamixel_to_joint_state.cpp.o: In function `XmlRpc::XmlRpcValue::operator bool&()':
/opt/ros/kinetic/include/XmlRpcValue.h:90: undefined reference to `XmlRpc::XmlRpcValue::assertTypeOrInvalid(XmlRpc::XmlRpcValue::Type)'
CMakeFiles/dynamixel_to_joint_state.dir/src/dynamixel_to_joint_state.cpp.o: In function `XmlRpc::XmlRpcValue::operator[](std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&)':
/opt/ros/kinetic/include/XmlRpcValue.h:100: undefined reference to `XmlRpc::XmlRpcValue::assertStruct()'
CMakeFiles/dynamixel_to_joint_state.dir/src/dynamixel_to_joint_state.cpp.o: In function `XmlRpc::XmlRpcValue::operator[](char const*)':
/opt/ros/kinetic/include/XmlRpcValue.h:101: undefined reference to `XmlRpc::XmlRpcValue::assertStruct()'
CMakeFiles/dynamixel_to_joint_state.dir/src/dynamixel_to_joint_state.cpp.o: In function `XmlRpc::XmlRpcValue::operator double&()':
/opt/ros/kinetic/include/XmlRpcValue.h:92: undefined reference to `XmlRpc::XmlRpcValue::assertTypeOrInvalid(XmlRpc::XmlRpcValue::Type)'
CMakeFiles/dynamixel_to_joint_state.dir/src/dynamixel_to_joint_state.cpp.o: In function `void ros::Publisher::publish<sensor_msgs::JointState_<std::allocator<void> > >(sensor_msgs::JointState_<std::allocator<void> > const&) const':
/opt/ros/kinetic/include/ros/publisher.h:106: undefined reference to `ros::Publisher::Impl::isValid() const'
/opt/ros/kinetic/include/ros/publisher.h:118: undefined reference to `ros::Publisher::publish(boost::function<ros::SerializedMessage ()> const&, ros::SerializedMessage&) const'
CMakeFiles/dynamixel_to_joint_state.dir/src/dynamixel_to_joint_state.cpp.o: In function `XmlRpc::XmlRpcValue::~XmlRpcValue()':
/opt/ros/kinetic/include/XmlRpcValue.h:76: undefined reference to `XmlRpc::XmlRpcValue::invalidate()'
CMakeFiles/dynamixel_to_joint_state.dir/src/dynamixel_to_joint_state.cpp.o: In function `ros::Publisher ros::NodeHandle::advertise<sensor_msgs::JointState_<std::allocator<void> > >(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&, unsigned int, bool)':
/opt/ros/kinetic/include/ros/node_handle.h:253: undefined reference to `ros::NodeHandle::advertise(ros::AdvertiseOptions&)'
CMakeFiles/dynamixel_to_joint_state.dir/src/dynamixel_to_joint_state.cpp.o: In function `XmlRpc::XmlRpcValue::~XmlRpcValue()':
/opt/ros/kinetic/include/XmlRpcValue.h:76: undefined reference to `XmlRpc::XmlRpcValue::invalidate()'
CMakeFiles/dynamixel_to_joint_state.dir/src/dynamixel_to_joint_state.cpp.o: In function `ros::Subscriber ros::NodeHandle::subscribe<boost::shared_ptr<dynamixel_msgs::MotorStateList_<std::allocator<void> > > const&>(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&, unsigned int, void (*)(boost::shared_ptr<dynamixel_msgs::MotorStateList_<std::allocator<void> > > const&), ros::TransportHints const&)':
/opt/ros/kinetic/include/ros/node_handle.h:660: undefined reference to `ros::NodeHandle::subscribe(ros::SubscribeOptions&)'
CMakeFiles/dynamixel_to_joint_state.dir/src/dynamixel_to_joint_state.cpp.o: In function `main':
/home/remy-xubuntu/catkin_a_price/src/reactor_controller/src/dynamixel_to_joint_state.cpp:96: undefined reference to `ros::init(int&, char**, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&, unsigned int)'
/home/remy-xubuntu/catkin_a_price/src/reactor_controller/src/dynamixel_to_joint_state.cpp:97: undefined reference to `ros::NodeHandle::NodeHandle(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&, std::map<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >, std::less<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > >, std::allocator<std::pair<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > > > > const&)'
/home/remy-xubuntu/catkin_a_price/src/reactor_controller/src/dynamixel_to_joint_state.cpp:105: undefined reference to `ros::NodeHandle::getParam(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >&) const'
/home/remy-xubuntu/catkin_a_price/src/reactor_controller/src/dynamixel_to_joint_state.cpp:107: undefined reference to `ros::console::g_initialized'
/home/remy-xubuntu/catkin_a_price/src/reactor_controller/src/dynamixel_to_joint_state.cpp:110: undefined reference to `urdf::Model::initString(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&)'
/home/remy-xubuntu/catkin_a_price/src/reactor_controller/src/dynamixel_to_joint_state.cpp:112: undefined reference to `ros::console::g_initialized'
/home/remy-xubuntu/catkin_a_price/src/reactor_controller/src/dynamixel_to_joint_state.cpp:115: undefined reference to `ros::NodeHandle::getParam(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&, XmlRpc::XmlRpcValue&) const'
/home/remy-xubuntu/catkin_a_price/src/reactor_controller/src/dynamixel_to_joint_state.cpp:117: undefined reference to `ros::console::g_initialized'
CMakeFiles/dynamixel_to_joint_state.dir/src/dynamixel_to_joint_state.cpp.o: In function `XmlRpc::XmlRpcValue::begin[abi:cxx11]()':
/opt/ros/kinetic/include/XmlRpcValue.h:103: undefined reference to `XmlRpc::XmlRpcValue::assertStruct()'
CMakeFiles/dynamixel_to_joint_state.dir/src/dynamixel_to_joint_state.cpp.o: In function `XmlRpc::XmlRpcValue::end[abi:cxx11]()':
/opt/ros/kinetic/include/XmlRpcValue.h:104: undefined reference to `XmlRpc::XmlRpcValue::assertStruct()'
CMakeFiles/dynamixel_to_joint_state.dir/src/dynamixel_to_joint_state.cpp.o: In function `XmlRpc::XmlRpcValue::XmlRpcValue(XmlRpc::XmlRpcValue const&)':
/opt/ros/kinetic/include/XmlRpcValue.h:73: undefined reference to `XmlRpc::XmlRpcValue::operator=(XmlRpc::XmlRpcValue const&)'
CMakeFiles/dynamixel_to_joint_state.dir/src/dynamixel_to_joint_state.cpp.o: In function `main':
/home/remy-xubuntu/catkin_a_price/src/reactor_controller/src/dynamixel_to_joint_state.cpp:125: undefined reference to `XmlRpc::XmlRpcValue::hasMember(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&) const'
CMakeFiles/dynamixel_to_joint_state.dir/src/dynamixel_to_joint_state.cpp.o: In function `XmlRpc::XmlRpcValue::operator[](char const*)':
/opt/ros/kinetic/include/XmlRpcValue.h:101: undefined reference to `XmlRpc::XmlRpcValue::assertStruct()'
CMakeFiles/dynamixel_to_joint_state.dir/src/dynamixel_to_joint_state.cpp.o: In function `XmlRpc::XmlRpcValue::operator std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >&[abi:cxx11]()':
/opt/ros/kinetic/include/XmlRpcValue.h:93: undefined reference to `XmlRpc::XmlRpcValue::assertTypeOrInvalid(XmlRpc::XmlRpcValue::Type)'
CMakeFiles/dynamixel_to_joint_state.dir/src/dynamixel_to_joint_state.cpp.o: In function `XmlRpc::XmlRpcValue::operator[](char const*)':
/opt/ros/kinetic/include/XmlRpcValue.h:101: undefined reference to `XmlRpc::XmlRpcValue::assertStruct()'
/opt/ros/kinetic/include/XmlRpcValue.h:101: undefined reference to `XmlRpc::XmlRpcValue::assertStruct()'
CMakeFiles/dynamixel_to_joint_state.dir/src/dynamixel_to_joint_state.cpp.o: In function `XmlRpc::XmlRpcValue::operator int&()':
/opt/ros/kinetic/include/XmlRpcValue.h:91: undefined reference to `XmlRpc::XmlRpcValue::assertTypeOrInvalid(XmlRpc::XmlRpcValue::Type)'
CMakeFiles/dynamixel_to_joint_state.dir/src/dynamixel_to_joint_state.cpp.o: In function `main':
/home/remy-xubuntu/catkin_a_price/src/reactor_controller/src/dynamixel_to_joint_state.cpp:129: undefined reference to `XmlRpc::XmlRpcValue::hasMember(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&) const'
CMakeFiles/dynamixel_to_joint_state.dir/src/dynamixel_to_joint_state.cpp.o: In function `XmlRpc::XmlRpcValue::~XmlRpcValue()':
/opt/ros/kinetic/include/XmlRpcValue.h:76: undefined reference to `XmlRpc::XmlRpcValue::invalidate()'
CMakeFiles/dynamixel_to_joint_state.dir/src/dynamixel_to_joint_state.cpp.o: In function `XmlRpc::XmlRpcValue::operator[](char const*)':
/opt/ros/kinetic/include/XmlRpcValue.h:101: undefined reference to `XmlRpc::XmlRpcValue::assertStruct()'
CMakeFiles/dynamixel_to_joint_state.dir/src/dynamixel_to_joint_state.cpp.o: In function `XmlRpc::XmlRpcValue::operator std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >&[abi:cxx11]()':
/opt/ros/kinetic/include/XmlRpcValue.h:93: undefined reference to `XmlRpc::XmlRpcValue::assertTypeOrInvalid(XmlRpc::XmlRpcValue::Type)'
CMakeFiles/dynamixel_to_joint_state.dir/src/dynamixel_to_joint_state.cpp.o: In function `XmlRpc::XmlRpcValue::operator[](char const*)':
/opt/ros/kinetic/include/XmlRpcValue.h:101: undefined reference to `XmlRpc::XmlRpcValue::assertStruct()'
/opt/ros/kinetic/include/XmlRpcValue.h:101: undefined reference to `XmlRpc::XmlRpcValue::assertStruct()'
CMakeFiles/dynamixel_to_joint_state.dir/src/dynamixel_to_joint_state.cpp.o: In function `XmlRpc::XmlRpcValue::operator int&()':
/opt/ros/kinetic/include/XmlRpcValue.h:91: undefined reference to `XmlRpc::XmlRpcValue::assertTypeOrInvalid(XmlRpc::XmlRpcValue::Type)'
CMakeFiles/dynamixel_to_joint_state.dir/src/dynamixel_to_joint_state.cpp.o: In function `main':
/home/remy-xubuntu/catkin_a_price/src/reactor_controller/src/dynamixel_to_joint_state.cpp:136: undefined reference to `ros::NodeHandle::getParam(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&, XmlRpc::XmlRpcValue&) const'
/home/remy-xubuntu/catkin_a_price/src/reactor_controller/src/dynamixel_to_joint_state.cpp:138: undefined reference to `ros::console::g_initialized'
/home/remy-xubuntu/catkin_a_price/src/reactor_controller/src/dynamixel_to_joint_state.cpp:141: undefined reference to `ros::Publisher::~Publisher()'
/home/remy-xubuntu/catkin_a_price/src/reactor_controller/src/dynamixel_to_joint_state.cpp:142: undefined reference to `ros::Subscriber::~Subscriber()'
/home/remy-xubuntu/catkin_a_price/src/reactor_controller/src/dynamixel_to_joint_state.cpp:144: undefined reference to `ros::spin()'
/home/remy-xubuntu/catkin_a_price/src/reactor_controller/src/dynamixel_to_joint_state.cpp:97: undefined reference to `ros::NodeHandle::~NodeHandle()'
/home/remy-xubuntu/catkin_a_price/src/reactor_controller/src/dynamixel_to_joint_state.cpp:112: undefined reference to `ros::console::setLogLocationLevel(ros::console::LogLocation*, ros::console::levels::Level)'
/home/remy-xubuntu/catkin_a_price/src/reactor_controller/src/dynamixel_to_joint_state.cpp:112: undefined reference to `ros::console::checkLogLocationEnabled(ros::console::LogLocation*)'
/home/remy-xubuntu/catkin_a_price/src/reactor_controller/src/dynamixel_to_joint_state.cpp:112: undefined reference to `ros::console::print(ros::console::FilterBase*, void*, ros::console::levels::Level, char const*, int, char const*, char const*, ...)'
/home/remy-xubuntu/catkin_a_price/src/reactor_controller/src/dynamixel_to_joint_state.cpp:117: undefined reference to `ros::console::initialize()'
/home/remy-xubuntu/catkin_a_price/src/reactor_controller/src/dynamixel_to_joint_state.cpp:117: undefined reference to `ros::console::initializeLogLocation(ros::console::LogLocation*, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&, ros::console::levels::Level)'
/home/remy-xubuntu/catkin_a_price/src/reactor_controller/src/dynamixel_to_joint_state.cpp:117: undefined reference to `ros::console::setLogLocationLevel(ros::console::LogLocation*, ros::console::levels::Level)'
/home/remy-xubuntu/catkin_a_price/src/reactor_controller/src/dynamixel_to_joint_state.cpp:117: undefined reference to `ros::console::checkLogLocationEnabled(ros::console::LogLocation*)'
/home/remy-xubuntu/catkin_a_price/src/reactor_controller/src/dynamixel_to_joint_state.cpp:117: undefined reference to `ros::console::print(ros::console::FilterBase*, void*, ros::console::levels::Level, char const*, int, char const*, char const*, ...)'
/home/remy-xubuntu/catkin_a_price/src/reactor_controller/src/dynamixel_to_joint_state.cpp:107: undefined reference to `ros::console::initialize()'
/home/remy-xubuntu/catkin_a_price/src/reactor_controller/src/dynamixel_to_joint_state.cpp:107: undefined reference to `ros::console::initializeLogLocation(ros::console::LogLocation*, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&, ros::console::levels::Level)'
/home/remy-xubuntu/catkin_a_price/src/reactor_controller/src/dynamixel_to_joint_state.cpp:107: undefined reference to `ros::console::setLogLocationLevel(ros::console::LogLocation*, ros::console::levels::Level)'
/home/remy-xubuntu/catkin_a_price/src/reactor_controller/src/dynamixel_to_joint_state.cpp:107: undefined reference to `ros::console::checkLogLocationEnabled(ros::console::LogLocation*)'
/home/remy-xubuntu/catkin_a_price/src/reactor_controller/src/dynamixel_to_joint_state.cpp:107: undefined reference to `ros::console::print(ros::console::FilterBase*, void*, ros::console::levels::Level, char const*, int, char const*, char const*, ...)'
/home/remy-xubuntu/catkin_a_price/src/reactor_controller/src/dynamixel_to_joint_state.cpp:112: undefined reference to `ros::console::initialize()'
/home/remy-xubuntu/catkin_a_price/src/reactor_controller/src/dynamixel_to_joint_state.cpp:112: undefined reference to `ros::console::initializeLogLocation(ros::console::LogLocation*, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&, ros::console::levels::Level)'
/home/remy-xubuntu/catkin_a_price/src/reactor_controller/src/dynamixel_to_joint_state.cpp:138: undefined reference to `ros::console::print(ros::console::FilterBase*, void*, ros::console::levels::Level, char const*, int, char const*, char const*, ...)'
/home/remy-xubuntu/catkin_a_price/src/reactor_controller/src/dynamixel_to_joint_state.cpp:138: undefined reference to `ros::console::initialize()'
/home/remy-xubuntu/catkin_a_price/src/reactor_controller/src/dynamixel_to_joint_state.cpp:138: undefined reference to `ros::console::initializeLogLocation(ros::console::LogLocation*, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&, ros::console::levels::Level)'
/home/remy-xubuntu/catkin_a_price/src/reactor_controller/src/dynamixel_to_joint_state.cpp:138: undefined reference to `ros::console::setLogLocationLevel(ros::console::LogLocation*, ros::console::levels::Level)'
/home/remy-xubuntu/catkin_a_price/src/reactor_controller/src/dynamixel_to_joint_state.cpp:138: undefined reference to `ros::console::checkLogLocationEnabled(ros::console::LogLocation*)'
/home/remy-xubuntu/catkin_a_price/src/reactor_controller/src/dynamixel_to_joint_state.cpp:97: undefined reference to `ros::NodeHandle::~NodeHandle()'
CMakeFiles/dynamixel_to_joint_state.dir/src/dynamixel_to_joint_state.cpp.o: In function `XmlRpc::XmlRpcValue::~XmlRpcValue()':
/opt/ros/kinetic/include/XmlRpcValue.h:76: undefined reference to `XmlRpc::XmlRpcValue::invalidate()'
CMakeFiles/dynamixel_to_joint_state.dir/src/dynamixel_to_joint_state.cpp.o: In function `__static_initialization_and_destruction_0':
/home/remy-xubuntu/catkin_a_price/src/reactor_controller/src/dynamixel_to_joint_state.cpp:44: undefined reference to `ros::Publisher::~Publisher()'
/home/remy-xubuntu/catkin_a_price/src/reactor_controller/src/dynamixel_to_joint_state.cpp:45: undefined reference to `ros::Subscriber::~Subscriber()'
CMakeFiles/dynamixel_to_joint_state.dir/src/dynamixel_to_joint_state.cpp.o: In function `ros::SubscriptionCallbackHelperT<boost::shared_ptr<dynamixel_msgs::MotorStateList_<std::allocator<void> > > const&, void>::deserialize(ros::SubscriptionCallbackHelperDeserializeParams const&)':
/opt/ros/kinetic/include/ros/subscription_callback_helper.h:126: undefined reference to `ros::console::g_initialized'
/opt/ros/kinetic/include/ros/subscription_callback_helper.h:126: undefined reference to `ros::console::initialize()'
/opt/ros/kinetic/include/ros/subscription_callback_helper.h:126: undefined reference to `ros::console::setLogLocationLevel(ros::console::LogLocation*, ros::console::levels::Level)'
/opt/ros/kinetic/include/ros/subscription_callback_helper.h:126: undefined reference to `ros::console::checkLogLocationEnabled(ros::console::LogLocation*)'
/opt/ros/kinetic/include/ros/subscription_callback_helper.h:126: undefined reference to `ros::console::initializeLogLocation(ros::console::LogLocation*, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&, ros::console::levels::Level)'
/opt/ros/kinetic/include/ros/subscription_callback_helper.h:126: undefined reference to `ros::console::print(ros::console::FilterBase*, void*, ros::console::levels::Level, char const*, int, char const*, char const*, ...)'
collect2: error: ld returned 1 exit status
reactor_controller/CMakeFiles/dynamixel_to_joint_state.dir/build.make:102: recipe for target '/home/remy-xubuntu/catkin_a_price/devel/lib/reactor_controller/dynamixel_to_joint_state' failed
make[2]: *** [/home/remy-xubuntu/catkin_a_price/devel/lib/reactor_controller/dynamixel_to_joint_state] Error 1
CMakeFiles/Makefile2:1817: recipe for target 'reactor_controller/CMakeFiles/dynamixel_to_joint_state.dir/all' failed
make[1]: *** [reactor_controller/CMakeFiles/dynamixel_to_joint_state.dir/all] Error 2
Makefile:138: recipe for target 'all' failed
make: *** [all] Error 2
Invoking "make -j4 -l4" failed
```

**Comment by a-price** [comment]
Was this issue also fixed by using the gazebo branch?

**Comment by Plzr** [comment]
It did not, alas. It might be the additional stuff I have in the package, however. I'm quite new to ROS, so there are definitely some things I did wrong in there. I guess don't worry about it unless someone else who also uses your stuff reports the same problem.

**Comment by a-price** [comment]
I just encountered this issue on a machine without MoveIt. Should be fixed by 07486ea.
